src/components/DocumentationTopic.vue

```diff
@@ -345,6 +349,22 @@ export default {
     };
   },
   computed: {
+    normalizedSwiftPath: ({
+      normalizePath,
+      swiftPath,
+    }) => (
+      swiftPath ? buildUrl(normalizePath(swiftPath), {
+        language: undefined,
+      }) : null
+    ),
```

Comment:
Wonder if this is enough 🤔 

```suggestion
    normalizedSwiftPath: ({
      normalizePath,
      swiftPath,
    }) => (
      swiftPath ? normalizePath(swiftPath) : null
    ),
```